tensorflow_addons/crf/README.md

```diff
@@ -0,0 +1,4 @@
+# Addons - Conditional Random Fields (CRF)
+
+
+## Standard API
```

Comment:
Github isn't making it easy to comment on empty files, so commenting here: if we don't need the python subdirs here (ie, crf/python/__init__.py), might be easier/clearer to remove throughout? Unless you feel it is helpful to have python/ versus cc/ for ops.

Reply:
IMO I like the separation between python vs. cc, but am open to removing it. As far as the entirely empty dirs (like crf), the init is just there so the directory structure can be checked into git.

Reply:
SGTM